.github/workflows/build_all.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: |
           pip3 install --upgrade pip
           pip3 install ansible fabric3 jsonpickle requests PyYAML
-          vagrant plugin install vagrant-vbguest vagrant-vbguest vagrant-mutate
+          vagrant plugin install vagrant-vbguest vagrant-disksize vagrant-vbguest vagrant-mutate
```

Comment:
Two times the same parameter `... vagrant-vbguest ... vagrant-vbguest ...` ?

Reply:
fixed